package.json

```diff
@@ -21,5 +21,8 @@
   "lint-staged": {
     "*.{js,jsx}": "eslint",
     "*.rb": "bundle exec --gemfile=playbook-website/Gemfile rubocop --parallel"
+  },
+  "resolutions": {
+    "playbook-ui": "portal:/Users/drew/Development/playbook/playbook"
```

Comment:
You will have to revert this in order to build in CI

Reply:
Not sure where this came from cause its not in my VS code
<img width="721" alt="Screen Shot 2022-05-05 at 11 46 15 AM" src="https://user-images.githubusercontent.com/84344530/166961998-09e469a4-e2e2-405e-8bcc-5c4db9333e25.png">